app/models/instance.rb

```diff
@@ -57,6 +57,8 @@ def find_tenant_by_host_or_default(host)
   #   Orders the instances by ID.
   scope :order_by_id, ->(direction = :asc) { order(id: direction) }
 
+  default_scope { order(name: :asc) }
```

Comment:
Better not use `default_scope` unless it's very necessary, especially for core models like `Instance` which are referenced everywhere.

Basically `default_scope` will appear everywhere (even in `joins`) and could give unexpected behavior. 
Some example after default scope:

`course.instance` => `SELECT instances. * FROM xxx ORDER BY instances.name`
`Instance.order(:id)` => `SELECT xxx ORDER BY instances.name" ASC, instances.id ASC`